src/lua/lua_util.c

```diff
@@ -1421,7 +1421,7 @@ lua_util_is_uppercase(lua_State *L)
 	LUA_TRACE_POINT;
 	int32_t i = 0;
 	UChar32 uc;
-	unsigned int nlc = 0, nuc = 0;
+	unsigned int nuc = 0, nlc = 0, noc = 0;
```

Comment:
In fact, these should be booleans in this case.

Reply:
Make sense.  I re-pushed.
Thanks for suggestion!